rubin_sim/maf/utils/astrometry_utils.py

```diff
@@ -42,36 +42,41 @@ def m52snr(m, m5):
 
     Parameters
     ----------
-    m : float or numpy.ndarray
+    m : `float` or `np.ndarray` (N,)
         The magnitude of the star
-    m5 : float or numpy.ndarray
+    m5 : `float` or `np.ndarray` (N,)
         The m5 limiting magnitude of the observation
 
     Returns
     -------
-    float or numpy.ndarray
+    snr : `float` or `np.ndarray` (N,)
         The SNR
     """
-    snr = 5.0 * 10.0 ** (-0.4 * (m - m5))
+    # gamma varies per band, but is fairly close to this value
+    rgamma = 0.039
+    xval = np.power(10, 0.4 * (m - m5))
+    snr = 1 / np.sqrt((0.04 - rgamma) * xval + rgamma * xval * xval)
     return snr
 
 
-def astrom_precision(fwhm, snr):
+def astrom_precision(fwhm, snr, systematic_floor=0.01):
```

Comment:
The proper motion and parallax metrics (and maybe others) are already adding a systematic error floor on their own. So those should be updated to use the new kwarg (or just default it to zero so it's backwards compatible).

Reply:
I'll set it to 0 so it's backwards compatible, but I did take the noise floor out of the parallax metric. Might not have properly done that with the proper motion though ..

Reply:
Ok, checked for other uses within MAF and updated those. 
However, it's true that others may be using this so will leave as default = 0.